### PR TITLE
Fix annotations and labels in Helm chart

### PR DESCRIPTION
**Pull request by adriangonz**
## Changelog
- Fix annotations and labels in `seldon-single-model` chart

### CODE REVIEW

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **adriangonz**

If they are not already assigned, you can assign the PR to them by writing `/assign @adriangonz` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by seldondev** [comment]
Wed Jun 17 11:33:43 UTC 2020
The logs for [pr-build] [1] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1967/1.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1967 --build=1

**Comment by seldondev** [comment]
Wed Jun 17 11:33:53 UTC 2020
The logs for [lint] [2] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1967/2.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1967 --build=2

**Comment by seldondev** [comment]
Wed Jun 17 11:36:28 UTC 2020
The logs for [notebooks] [3] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1967/3.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1967 --build=3

**Comment by seldondev** [comment]
Wed Jun 17 11:36:34 UTC 2020
The logs for [integration] [4] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1967/4.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1967 --build=4

File: helm-charts/seldon-single-model/templates/seldondeployment.json [hunk 1]
```diff
@@ -6,15 +6,15 @@
     "namespace": "{{ .Release.Namespace }}",
     "labels": {
       {{- range $name, $val := .Values.labels }}
-        "{{ $name }}": "{{ $val }}"
+        "{{ $name }}": "{{ $val }}",
```
Comment (axsaucedo) [inline helm-charts/seldon-single-model/templates/seldondeployment.json:9]: Looks good, however with javascript wouldn't you still have an issue if you have a trailing slash? I have seen that with arrays you can use {{if ne $index 0}},{{end}} but in this case you only have name,val so not sure whether it'd be the case. Did it not error out in your case with the extra comma at the end?

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/SeldonIO/seldon-core/pull/1967#issuecomment-645340250" title="Approved">axsaucedo</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)~~ [axsaucedo]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by seldondev** [comment]
@adriangonz: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
notebooks | 87e17b38602cbb1b63416ae8138e7dda405029fc | [link]() | `/test notebooks`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: helm-charts/seldon-single-model/templates/seldondeployment.json (hunk 1 above)
Comment (adriangonz) [reply helm-charts/seldon-single-model/templates/seldondeployment.json:9]: Helm usually reads and writes YAMLs, and it seems that the YAML parser is happy with trailing commas.

**Comment by seldondev** [comment]
failed to trigger Pull Request pipeline
* failed to create agent
* failed to calculate in repo config
* failed to load trigger config for repository SeldonIO/seldon-core for ref 87e17b38602cbb1b63416ae8138e7dda405029fc
* failed to find any lighthouse configuration files in repo SeldonIO/seldon-core at sha 87e17b38602cbb1b63416ae8138e7dda405029fc
* failed to process repo SeldonIO/seldon-core refref 87e17b38602cbb1b63416ae8138e7dda405029fc
* failed to list files in directory /var/tmp/gitrepo660600678/.lighthouse
* open /var/tmp/gitrepo660600678/.lighthouse
* no such file or directory

**Pull request merged**